sandbox/App.jsx

```diff
@@ -1,14 +1,24 @@
 import React from 'react';
 
-import { Renderer } from '@prezly/content-renderer-react-js';
+import { Renderer, RendererContextProvider } from '@prezly/content-renderer-react-js';
+import { STORY_BOOKMARK_NODE_TYPE } from '@prezly/slate-types';
 
-import '@prezly/content-renderer-react-js/styles.css'
+import '@prezly/content-renderer-react-js/styles.css';
 import './styles.css';
 
 import story from './story.json';
+import storySample from './storySample.json';
```

Comment:
> `story`
> `storySample`

I suggest finding better names for these, as now they sound synonymous. 

How about `content` and `referencedStory`?